src/annotator/config/test/settings-test.js

```diff
@@ -22,245 +27,42 @@ describe('annotator/config/settingsFrom', () => {
     $imports.$restore();
   });
 
-  describe('app frame URLs from link tags', () => {
-    function appendLink(href, rel) {
-      const link = document.createElement('link');
-      link.type = 'application/annotator+html';
-      link.rel = rel;
-      if (href) {
-        link.href = href;
-      }
-      document.head.appendChild(link);
-      return link;
-    }
-    describe('#notebookAppUrl', () => {
-      context(
-        "when there's an application/annotator+html notebook link",
-        () => {
-          let link;
-
-          beforeEach(
-            'add an application/annotator+html notebook <link>',
-            () => {
-              link = appendLink('http://example.com/app.html', 'notebook');
-            }
-          );
-
-          afterEach('tidy up the notebook link', () => {
-            link.remove();
-          });
-
-          it('returns the href from the notebook link', () => {
-            assert.equal(
-              settingsFrom(window).notebookAppUrl,
-              'http://example.com/app.html'
-            );
-          });
-        }
+  describe('#notebookAppUrl', () => {
+    it('calls urlFromLinkTag with appropriate params', () => {
+      assert.exists(settingsFrom(window).notebookAppUrl);
```

Comment:
I'm not sure it needs to because it's moved to a different module. From the standpoint of this module, my assumption was it just needs to make sure it calls that notebookAppUrl with the appropriate params. url-from-link-tag-test.js is where the specific values are tested. Does that seem reasonable?